content/en/real_user_monitoring/browser/collecting_browser_errors.md

```diff
@@ -140,6 +141,84 @@ try {
 {{% /tab %}}
 {{< /tabs >}}
 
+### React error boundaries instrumentation 
+
+You can instrument the React error boundaries to monitor React rendering errors using the RUM Browser SDK `addError()` API.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about adding a link to react error boundaries doc?

Reply:
Added, thanks for the suggestions!